engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/AuthenticationConfiguration.scala

```diff
@@ -1,21 +1,15 @@
 package pl.touk.nussknacker.ui.security.api
 
-import java.io.File
-import java.net.{URI, URL}
-import java.security.PublicKey
+import java.net.URI
 
-import com.typesafe.config.{Config, ConfigFactory}
-import pl.touk.nussknacker.engine.util.cache.CacheConfig
+import com.typesafe.config.Config
 import pl.touk.nussknacker.engine.util.config.ConfigFactoryExt
 import pl.touk.nussknacker.ui.security.api.AuthenticationConfiguration.{ConfigRule, ConfigUser}
-import pl.touk.nussknacker.ui.security.api.AuthenticationMethod.AuthenticationMethod
 import pl.touk.nussknacker.ui.security.api.GlobalPermission.GlobalPermission
 import pl.touk.nussknacker.ui.security.api.Permission.Permission
 
-import scala.concurrent.duration._
-
 trait AuthenticationConfiguration {
-  def method: AuthenticationMethod
+  def name: String
```

Comment:
I think we don't need it at all now. We only use name from AuthenticationProvider?

Reply:
We cannot use the name from an AuthenticationProvider directly, because we need the name value in an AuthenticationResources instance and it is the AuthenticationProvider that creates AuthenticationResources and not the other way round. Instead, we could use a constant like `BasicAuthenticationProvider.name` there. Would it be better?